src/components/BlacklistedTagsBadge/index.tsx

```diff
@@ -0,0 +1,60 @@
+import Badge from '@app/components/Common/Badge';
+import Tooltip from '@app/components/Common/Tooltip';
+import defineMessages from '@app/utils/defineMessages';
+import { TagIcon } from '@heroicons/react/20/solid';
+import type { BlacklistItem } from '@server/interfaces/api/blacklistInterfaces';
+import type { Keyword } from '@server/models/common';
+import { useEffect, useState } from 'react';
+import { useIntl } from 'react-intl';
+
+const messages = defineMessages('components.Settings', {
+  blacklistedTagsText: 'Blacklisted Tags',
+});
+
+interface BlacklistedTagsBadgeProps {
+  data: BlacklistItem;
+}
+
+const BlacklistedTagsBadge = ({ data }: BlacklistedTagsBadgeProps) => {
+  const [tagNamesBlacklistedFor, setTagNamesBlacklistedFor] =
+    useState<string>('Loading...');
+  const intl = useIntl();
+
+  useEffect(() => {
+    if (data.blacklistedTags == null) {
```

Comment:
Please use `===`

Reply:
I use == here to check for both null and undefined, instead of only null. It shouldn't matter, but it keeps things safe in case it's undefined for some reason

Reply:
Done! Became `if (!data.blacklistedTags) {`